src/utils/redux/migrations.ts

```diff
@@ -29,4 +30,20 @@ export const accountsMigrations = {
         }
       });
     }),
+  3: identity,
+} as any;
+
+export const beaconMigrations = {
+  0: identity,
+  1: identity,
+  2: identity,
+  3: (state: any) =>
+    produce(state, (draft: any) => {
```

Comment:
I remember I sent a cleaner script for this one 🤔

Reply:
Ah, shoot - it was overwritten by a draft that I had 😅

Reply:
nice

```
Last login: Wed Mar  6 11:57:41 on ttys003
➜  umami-v2 git:(fix-beacon-modal-opening) ✗ gst
On branch fix-beacon-modal-opening
Your branch is up to date with 'origin/fix-beacon-modal-opening'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   src/utils/beacon/useHandleBeaconMessage.test.tsx
	modified:   src/utils/beacon/useHandleBeaconMessage.tsx

no changes added to commit (use "git add" and/or "git commit -a")
➜  umami-v2 git:(fix-beacon-modal-opening) ✗ gd
➜  umami-v2 git:(fix-beacon-modal-opening) ✗ gc -a -m 'Open a batch sign page when >1 operations'
[fix-beacon-modal-opening 382b0421] Open a batch sign page when >1 operations
 2 files changed, 56 insertions(+), 2 deletions(-)
➜  umami-v2 git:(fix-beacon-modal-opening) gst
On branch fix-beacon-modal-opening
Your branch is ahead of 'origin/fix-beacon-modal-opening' by 1 commit.
  (use "git push" to publish your local commits)

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   src/components/AccountDrawer/JsValueWrap.tsx
	modified:   src/components/SendFlow/Beacon/BatchSignPage.tsx
	modified:   src/utils/beacon/useHandleBeaconMessage.tsx

no changes added to commit (use "git add" and/or "git commit -a")
➜  umami-v2 git:(fix-beacon-modal-opening) ✗ gd
➜  umami-v2 git:(fix-beacon-modal-opening) ✗ gco useHandleBeaconMessage.tsx
error: pathspec 'useHandleBeaconMessage.tsx' did not match any file(s) known to git
➜  umami-v2 git:(fix-beacon-modal-opening) ✗ gst
On branch fix-beacon-modal-opening
Your branch is ahead of 'origin/fix-beacon-modal-opening' by 1 commit.
  (use "git push" to publish your local commits)

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   src/components/AccountDrawer/JsValueWrap.tsx
	modified:   src/components/SendFlow/Beacon/BatchSignPage.tsx
	modified:   src/utils/beacon/useHandleBeaconMessage.tsx

no changes added to commit (use "git add" and/or "git commit -a")
➜  umami-v2 git:(fix-beacon-modal-opening) ✗ gd
➜  umami-v2 git:(fix-beacon-modal-opening) ✗ gco src/utils/beacon/useHandleBeaconMessage.tsx
Updated 1 path from the index
➜  umami-v2 git:(fix-beacon-modal-opening) ✗ gst
On branch fix-beacon-modal-opening
Your branch is ahead of 'origin/fix-beacon-modal-opening' by 1 commit.
  (use "git push" to publish your local commits)

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   src/components/AccountDrawer/JsValueWrap.tsx
	modified:   src/components/SendFlow/Beacon/BatchSignPage.tsx

no changes added to commit (use "git add" and/or "git commit -a")
➜  umami-v2 git:(fix-beacon-modal-opening) ✗ gd
➜  umami-v2 git:(fix-beacon-modal-opening) ✗ gc -a -m 'Add styling for Beacon BatchSignPage'
[fix-beacon-modal-opening d86eada6] Add styling for Beacon BatchSignPage
 2 files changed, 12 insertions(+), 8 deletions(-)
➜  umami-v2 git:(fix-beacon-modal-opening) gd
➜  umami-v2 git:(fix-beacon-modal-opening) gst
On branch fix-beacon-modal-opening
Your branch is ahead of 'origin/fix-beacon-modal-opening' by 2 commits.
  (use "git push" to publish your local commits)

nothing to commit, working tree clean
➜  umami-v2 git:(fix-beacon-modal-opening) gst
On branch fix-beacon-modal-opening
Your branch is ahead of 'origin/fix-beacon-modal-opening' by 2 commits.
  (use "git push" to publish your local commits)

nothing to commit, working tree clean
➜  umami-v2 git:(fix-beacon-modal-opening) gp 
Enumerating objects: 28, done.
Counting objects: 100% (28/28), done.
Delta compression using up to 10 threads
Compressing objects: 100% (16/16), done.
Writing objects: 100% (16/16), 1.61 KiB | 1.61 MiB/s, done.
Total 16 (delta 14), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (14/14), completed with 12 local objects.
To github.com-serjonya-trili:trilitech/umami-v2.git
   eb9de26e..d86eada6  fix-beacon-modal-opening -> fix-beacon-modal-opening
➜  umami-v2 git:(fix-beacon-modal-opening) yarn build
Creating an optimized production build...
Compiled successfully.

File sizes after gzip:

  793.72 kB (+551 B)  build/static/js/main.fbf26135.js
  2.94 kB             build/static/js/reactPlayerFilePlayer.82c993af.chunk.js
  1.91 kB             build/static/js/reactPlayerYouTube.de047225.chunk.js
  1.81 kB             build/static/js/reactPlayerMux.244822da.chunk.js
  1.41 kB             build/static/css/main.4ada0dcb.css
  1.39 kB             build/static/js/reactPlayerWistia.e46ee45f.chunk.js
  1.39 kB             build/static/js/reactPlayerVimeo.e403c288.chunk.js
  1.33 kB (+25 B)     build/static/js/reactPlayerPreview.ae77c75e.chunk.js
  1.32 kB             build/static/js/reactPlayerFacebook.5311619d.chunk.js
  1.28 kB             build/static/js/reactPlayerTwitch.20a64de5.chunk.js
  1.26 kB             build/static/js/reactPlayerDailyMotion.141330e7.chunk.js
  1.25 kB             build/static/js/reactPlayerSoundCloud.fca705a3.chunk.js
  1.22 kB             build/static/js/reactPlayerStreamable.962271bf.chunk.js
  1.21 kB             build/static/js/reactPlayerVidyard.c13e88aa.chunk.js
  1.2 kB              build/static/js/reactPlayerKaltura.1c0fb203.chunk.js
  1.16 kB             build/static/js/reactPlayerMixcloud.f6bc528b.chunk.js

The bundle size is significantly larger than recommended.
Consider reducing it with code splitting: https://goo.gl/9VhYWB
You can also analyze the project dependencies: https://goo.gl/LeUzfb

The project was built assuming it is hosted at ./.
You can control this with the homepage field in your package.json.

The build folder is ready to be deployed.

Find out more about deployment here:

  https://cra.link/deployment

➜  umami-v2 git:(fix-beacon-modal-opening) gst
On branch fix-beacon-modal-opening
Your branch is up to date with 'origin/fix-beacon-modal-opening'.

nothing to commit, working tree clean
➜  umami-v2 git:(fix-beacon-modal-opening) gst
On branch fix-beacon-modal-opening
Your branch is up to date with 'origin/fix-beacon-modal-opening'.

nothing to commit, working tree clean
➜  umami-v2 git:(fix-beacon-modal-opening) yarn format
.github/dependabot.yml 12ms (unchanged)
.github/pull_request_template.md 14ms (unchanged)
.github/workflows/build.yaml 7ms (unchanged)
.github/workflows/deploy-to-github-pages.yaml 3ms (unchanged)
.github/workflows/e2e.yaml 4ms (unchanged)
.github/workflows/test.yaml 2ms (unchanged)
.prettierrc 17ms (unchanged)
.yarnrc.yml 1ms (unchanged)
config/env.js 18ms (unchanged)
config/getHttpsConfig.js 5ms (unchanged)
config/jest/babelTransform.js 2ms (unchanged)
config/jest/cssTransform.js 1ms (unchanged)
config/jest/fileTransform.js 1ms (unchanged)
config/modules.js 5ms (unchanged)
config/paths.js 4ms (unchanged)
config/webpack.config.js 25ms (unchanged)
config/webpack/persistentCache/createEnvironmentHash.js 1ms (unchanged)
config/webpackDevServer.config.js 4ms (unchanged)
cucumber.cjs 1ms (unchanged)
docker-compose.yaml 4ms (unchanged)
package.json 2ms (unchanged)
public/electron.js 6ms (unchanged)
public/index.html 14ms (unchanged)
public/manifest.json 1ms (unchanged)
public/preload.js 2ms (unchanged)
README.md 13ms (unchanged)
scripts/build.js 7ms (unchanged)
scripts/start.js 5ms (unchanged)
scripts/test.js 2ms (unchanged)
src/assets/icons/Accounts.tsx 79ms (unchanged)
src/assets/icons/AddAccount.tsx 4ms (unchanged)
src/assets/icons/AddContact.tsx 2ms (unchanged)
src/assets/icons/AddressBook.tsx 2ms (unchanged)
src/assets/icons/Baker.tsx 4ms (unchanged)
src/assets/icons/Batch.tsx 1ms (unchanged)
src/assets/icons/BurgerMenu.tsx 1ms (unchanged)
src/assets/icons/CheckIcon.tsx 1ms (unchanged)
src/assets/icons/Checkmark.tsx 1ms (unchanged)
src/assets/icons/ChevronDown.tsx 1ms (unchanged)
src/assets/icons/ChevronRight.tsx 1ms (unchanged)
src/assets/icons/ChevronUp.tsx 1ms (unchanged)
src/assets/icons/Coin.tsx 2ms (unchanged)
src/assets/icons/Contact.tsx 1ms (unchanged)
src/assets/icons/Contract.tsx 1ms (unchanged)
src/assets/icons/CrossedCircle.tsx 1ms (unchanged)
src/assets/icons/Diamond.tsx 2ms (unchanged)
src/assets/icons/Document.tsx 1ms (unchanged)
src/assets/icons/DoubleCheckmark.tsx 1ms (unchanged)
src/assets/icons/Download.tsx 1ms (unchanged)
src/assets/icons/EditAccount.tsx 1ms (unchanged)
src/assets/icons/Exclamation.tsx 2ms (unchanged)
src/assets/icons/ExitArrow.tsx 1ms (unchanged)
src/assets/icons/ExternalLink.tsx 1ms (unchanged)
src/assets/icons/Eye.tsx 2ms (unchanged)
src/assets/icons/EyeSlash.tsx 1ms (unchanged)
src/assets/icons/FA1.2.tsx 1ms (unchanged)
src/assets/icons/FA2.tsx 1ms (unchanged)
src/assets/icons/Feedback.tsx 1ms (unchanged)
src/assets/icons/Fetching.tsx 1ms (unchanged)
src/assets/icons/FileArrowDown.tsx 1ms (unchanged)
src/assets/icons/FileCopy.tsx 1ms (unchanged)
src/assets/icons/FlipForwardEnergy.tsx 1ms (unchanged)
src/assets/icons/Gear.tsx 1ms (unchanged)
src/assets/icons/Help.tsx 1ms (unchanged)
src/assets/icons/Hourglass.tsx 1ms (unchanged)
src/assets/icons/IncomingArrow.tsx 2ms (unchanged)
src/assets/icons/index.tsx 2ms (unchanged)
src/assets/icons/Key.tsx 2ms (unchanged)
src/assets/icons/Ledger.tsx 1ms (unchanged)
src/assets/icons/Link.tsx 1ms (unchanged)
src/assets/icons/Lock.tsx 1ms (unchanged)
src/assets/icons/Maintenance.tsx 1ms (unchanged)
src/assets/icons/Maki.tsx 2ms (unchanged)
src/assets/icons/Notice.tsx 1ms (unchanged)
src/assets/icons/OutgoingArrow.tsx 1ms (unchanged)
src/assets/icons/Pen.tsx 1ms (unchanged)
src/assets/icons/Plus.tsx 1ms (unchanged)
src/assets/icons/RefreshClock.tsx 2ms (unchanged)
src/assets/icons/Rotate.tsx 1ms (unchanged)
src/assets/icons/Slash.tsx 1ms (unchanged)
src/assets/icons/Sliders.tsx 1ms (unchanged)
src/assets/icons/Social.tsx 2ms (unchanged)
src/assets/icons/Tez.tsx 2ms (unchanged)
src/assets/icons/Token.tsx 2ms (unchanged)
src/assets/icons/Trash.tsx 2ms (unchanged)
src/assets/icons/UnknownContact.tsx 1ms (unchanged)
src/assets/icons/USB.tsx 1ms (unchanged)
src/assets/icons/Verified.tsx 2ms (unchanged)
src/assets/icons/WalletPlus.tsx 0ms (unchanged)
src/assets/icons/Warning.tsx 1ms (unchanged)
src/assets/icons/WindowLink.tsx 1ms (unchanged)
src/assets/icons/XMark.tsx 1ms (unchanged)
src/components/AccountBalance.test.tsx 2ms (unchanged)
src/components/AccountBalance.tsx 2ms (unchanged)
src/components/AccountDrawer/AccountDrawerDisplay.test.tsx 24ms (unchanged)
src/components/AccountDrawer/AccountDrawerDisplay.tsx 8ms (unchanged)
src/components/AccountDrawer/AssetsPanel/AssetPanel.test.tsx 2ms (unchanged)
src/components/AccountDrawer/AssetsPanel/AssetsPanel.tsx 6ms (unchanged)
src/components/AccountDrawer/AssetsPanel/DelegationDisplay.test.tsx 7ms (unchanged)
src/components/AccountDrawer/AssetsPanel/DelegationDisplay.tsx 5ms (unchanged)
src/components/AccountDrawer/AssetsPanel/MultisigPendingOperations/index.tsx 1ms (unchanged)
src/components/AccountDrawer/AssetsPanel/MultisigPendingOperations/MultisigActionButton.test.tsx 5ms (unchanged)
src/components/AccountDrawer/AssetsPanel/MultisigPendingOperations/MultisigActionButton.tsx 3ms (unchanged)
src/components/AccountDrawer/AssetsPanel/MultisigPendingOperations/MultisigDecodedOperation.test.tsx 5ms (unchanged)
src/components/AccountDrawer/AssetsPanel/MultisigPendingOperations/MultisigDecodedOperation.tsx 5ms (unchanged)
src/components/AccountDrawer/AssetsPanel/MultisigPendingOperations/MultisigDecodedOperations.tsx 3ms (unchanged)
src/components/AccountDrawer/AssetsPanel/MultisigPendingOperations/MultisigPendingOperation.test.tsx 6ms (unchanged)
src/components/AccountDrawer/AssetsPanel/MultisigPendingOperations/MultisigPendingOperation.tsx 2ms (unchanged)
src/components/AccountDrawer/AssetsPanel/MultisigPendingOperations/MultisigPendingOperations.test.tsx 3ms (unchanged)
src/components/AccountDrawer/AssetsPanel/MultisigPendingOperations/MultisigSignerTile.test.tsx 3ms (unchanged)
src/components/AccountDrawer/AssetsPanel/MultisigPendingOperations/MultisigSignerTile.tsx 4ms (unchanged)
src/components/AccountDrawer/AssetsPanel/NFTsGrid.test.tsx 6ms (unchanged)
src/components/AccountDrawer/AssetsPanel/NFTsGrid.tsx 5ms (unchanged)
src/components/AccountDrawer/AssetsPanel/OperationListDisplay.test.tsx 4ms (unchanged)
src/components/AccountDrawer/AssetsPanel/OperationListDisplay.tsx 2ms (unchanged)
src/components/AccountDrawer/AssetsPanel/TokenList.test.tsx 5ms (unchanged)
src/components/AccountDrawer/AssetsPanel/TokenList.tsx 3ms (unchanged)
src/components/AccountDrawer/AssetsPanel/ViewAllLink.test.tsx 1ms (unchanged)
src/components/AccountDrawer/AssetsPanel/ViewAllLink.tsx 2ms (unchanged)
src/components/AccountDrawer/index.tsx 2ms (unchanged)
src/components/AccountDrawer/JsValueWrap.tsx 1ms (unchanged)
src/components/AccountDrawer/MultisigApprovers.tsx 2ms (unchanged)
src/components/AccountDrawer/RenameAccount.test.tsx 3ms (unchanged)
src/components/AccountDrawer/RenameAccountModal.tsx 3ms (unchanged)
src/components/AccountDrawer/RenameRemoveMenuSwitch.test.tsx 4ms (unchanged)
src/components/AccountDrawer/RenameRemoveMenuSwitch.tsx 2ms (unchanged)
src/components/AccountSelector/AccountListDisplay.tsx 2ms (unchanged)
src/components/AccountTile/AccountTile.test.tsx 10ms (unchanged)
src/components/AccountTile/AccountTile.tsx 8ms (unchanged)
src/components/AccountTile/AccountTileIcon.test.tsx 3ms (unchanged)
src/components/AccountTile/AccountTileIcon.tsx 1ms (unchanged)
src/components/AddressAutocomplete/AddressAutocomplete.test.tsx 10ms (unchanged)
src/components/AddressAutocomplete/AddressAutocomplete.tsx 10ms (unchanged)
src/components/AddressAutocomplete/index.tsx 1ms (unchanged)
src/components/AddressAutocomplete/Suggestions.tsx 2ms (unchanged)
src/components/AddressPill/AddressPill.test.tsx 3ms (unchanged)
src/components/AddressPill/AddressPill.tsx 4ms (unchanged)
src/components/AddressPill/AddressPillIcon.test.tsx 2ms (unchanged)
src/components/AddressPill/AddressPillIcon.tsx 2ms (unchanged)
src/components/AddressPill/AddressPillText.test.tsx 2ms (unchanged)
src/components/AddressPill/AddressPillText.tsx 1ms (unchanged)
src/components/AddressPill/types.ts 1ms (unchanged)
src/components/AddressPill/useAddressKind.test.ts 6ms (unchanged)
src/components/AddressPill/useAddressKind.ts 2ms (unchanged)
src/components/AddressTile/AddressTile.test.tsx 3ms (unchanged)
src/components/AddressTile/AddressTile.tsx 2ms (unchanged)
src/components/AddressTile/AddressTileIcon.test.tsx 2ms (unchanged)
src/components/AddressTile/AddressTileIcon.tsx 2ms (unchanged)
src/components/AddressTile/types.ts 2ms (unchanged)
src/components/AddressTile/useAddressKind.test.ts 4ms (unchanged)
src/components/AddressTile/useAddressKind.ts 1ms (unchanged)
src/components/AnnouncementBanner.test.tsx 2ms (unchanged)
src/components/AnnouncementBanner.tsx 2ms (unchanged)
src/components/AppVersion.tsx 1ms (unchanged)
src/components/AssetTiles/TezTile.test.tsx 1ms (unchanged)
src/components/AssetTiles/TezTile.tsx 1ms (unchanged)
src/components/BuyTez/BuyTezForm.test.tsx 1ms (unchanged)
src/components/BuyTez/BuyTezForm.tsx 2ms (unchanged)
src/components/ChangePassword/ChangePasswordForm.test.tsx 4ms (unchanged)
src/components/ChangePassword/ChangePasswordForm.tsx 3ms (unchanged)
src/components/CircleIcon.tsx 1ms (unchanged)
src/components/ClickableCard.tsx 2ms (unchanged)
src/components/CollapseMenuButton/CollapseMenuButton.tsx 1ms (unchanged)
src/components/CollapseMenuButton/index.tsx 0ms (unchanged)
src/components/CollapseMenuButton/useCollapseMenu.test.ts 1ms (unchanged)
src/components/CollapseMenuButton/useCollapseMenu.ts 1ms (unchanged)
src/components/ConfirmationModal.test.tsx 2ms (unchanged)
src/components/ConfirmationModal.tsx 1ms (unchanged)
src/components/ContactModal.tsx 5ms (unchanged)
src/components/CopyableText.tsx 3ms (unchanged)
src/components/CSVFileUploader/CSVFileUploadForm.tsx 5ms (unchanged)
src/components/CSVFileUploader/index.tsx 1ms (unchanged)
src/components/CSVFileUploader/utils.test.ts 4ms (unchanged)
src/components/CSVFileUploader/utils.ts 2ms (unchanged)
src/components/DynamicModal.test.tsx 3ms (unchanged)
src/components/DynamicModal.tsx 3ms (unchanged)
src/components/ErrorPage.tsx 2ms (unchanged)
src/components/ExternalLink.tsx 1ms (unchanged)
src/components/FormErrorMessage.tsx 1ms (unchanged)
src/components/Identicon.tsx 1ms (unchanged)
src/components/MakiLogo.tsx 1ms (unchanged)
src/components/MnemonicAutocomplete.test.tsx 5ms (unchanged)
src/components/MnemonicAutocomplete.tsx 2ms (unchanged)
src/components/ModalBackButton.tsx 1ms (unchanged)
src/components/NestedScroll.tsx 1ms (unchanged)
src/components/NetworkSelector.test.tsx 2ms (unchanged)
src/components/NetworkSelector.tsx 2ms (unchanged)
src/components/NoItems/index.tsx 3ms (unchanged)
src/components/Offboarding/OffboardingForm.tsx 3ms (unchanged)
src/components/Offboarding/useOffboardingModal.tsx 1ms (unchanged)
src/components/Onboarding/connectOptions/ConnectOptions.test.tsx 3ms (unchanged)
src/components/Onboarding/connectOptions/ConnectOptions.tsx 2ms (unchanged)
src/components/Onboarding/connectOrCreate/ConnectOrCreate.test.tsx 2ms (unchanged)
src/components/Onboarding/connectOrCreate/ConnectOrCreate.tsx 2ms (unchanged)
src/components/Onboarding/derivationPath/DerivationPath.test.tsx 3ms (unchanged)
src/components/Onboarding/derivationPath/DerivationPath.tsx 3ms (unchanged)
src/components/Onboarding/eula/Eula.test.tsx 2ms (unchanged)
src/components/Onboarding/eula/Eula.tsx 2ms (unchanged)
src/components/Onboarding/FakeAccount.tsx 3ms (unchanged)
src/components/Onboarding/masterPassword/MasterPassword.test.tsx 1ms (unchanged)
src/components/Onboarding/masterPassword/MasterPassword.tsx 2ms (unchanged)
src/components/Onboarding/masterPassword/password/EnterAndConfirmPassword.test.tsx 2ms (unchanged)
src/components/Onboarding/masterPassword/password/EnterAndConfirmPassword.tsx 2ms (unchanged)
src/components/Onboarding/masterPassword/password/EnterPassword.test.tsx 2ms (unchanged)
src/components/Onboarding/masterPassword/password/EnterPassword.tsx 2ms (unchanged)
src/components/Onboarding/ModalContentWrapper.tsx 1ms (unchanged)
src/components/Onboarding/nameAccount/NameAccount.test.tsx 6ms (unchanged)
src/components/Onboarding/nameAccount/NameAccount.tsx 2ms (unchanged)
src/components/Onboarding/nameAccount/NameAccountDisplay.tsx 1ms (unchanged)
src/components/Onboarding/notice/Notice.test.tsx 1ms (unchanged)
src/components/Onboarding/notice/Notice.tsx 2ms (unchanged)
src/components/Onboarding/OnboardingStep.tsx 2ms (unchanged)
src/components/Onboarding/restoreBackupFile/RestoreBackupFile.test.tsx 9ms (unchanged)
src/components/Onboarding/restoreBackupFile/RestoreBackupFile.tsx 4ms (unchanged)
src/components/Onboarding/restoreBackupFile/utils.ts 4ms (unchanged)
src/components/Onboarding/restoreLedger/RestoreLedger.test.tsx 2ms (unchanged)
src/components/Onboarding/restoreLedger/RestoreLedger.tsx 3ms (unchanged)
src/components/Onboarding/restoreMnemonic/RestoreMnemonic.test.tsx 4ms (unchanged)
src/components/Onboarding/restoreMnemonic/RestoreMnemonic.tsx 6ms (unchanged)
src/components/Onboarding/restoreSecretKey/RestoreSecretKey.test.tsx 5ms (unchanged)
src/components/Onboarding/restoreSecretKey/RestoreSecretKey.tsx 3ms (unchanged)
src/components/Onboarding/showSeedphrase/ShowSeedphrase.test.tsx 2ms (unchanged)
src/components/Onboarding/showSeedphrase/ShowSeedphrase.tsx 2ms (unchanged)
src/components/Onboarding/useOnboardingModal.tsx 3ms (unchanged)
src/components/Onboarding/verifySeedphrase/VerifySeedphrase.test.tsx 3ms (unchanged)
src/components/Onboarding/verifySeedphrase/VerifySeedphrase.tsx 3ms (unchanged)
src/components/OperationTile/ContractCallTile.test.tsx 4ms (unchanged)
src/components/OperationTile/ContractCallTile.tsx 2ms (unchanged)
src/components/OperationTile/DelegationTile.test.tsx 4ms (unchanged)
src/components/OperationTile/DelegationTile.tsx 2ms (unchanged)
src/components/OperationTile/Fee.test.tsx 2ms (unchanged)
src/components/OperationTile/Fee.tsx 2ms (unchanged)
src/components/OperationTile/index.ts 0ms (unchanged)
src/components/OperationTile/OperationStatus.test.tsx 2ms (unchanged)
src/components/OperationTile/OperationStatus.tsx 1ms (unchanged)
src/components/OperationTile/OperationTile.test.tsx 3ms (unchanged)
src/components/OperationTile/OperationTile.tsx 2ms (unchanged)
src/components/OperationTile/OperationTileContext.ts 1ms (unchanged)
src/components/OperationTile/OperationTypeWrapper.test.tsx 1ms (unchanged)
src/components/OperationTile/OperationTypeWrapper.tsx 1ms (unchanged)
src/components/OperationTile/OriginationTile.test.tsx 4ms (unchanged)
src/components/OperationTile/OriginationTile.tsx 3ms (unchanged)
src/components/OperationTile/testUtils.ts 2ms (unchanged)
src/components/OperationTile/Timestamp.test.tsx 2ms (unchanged)
src/components/OperationTile/Timestamp.tsx 1ms (unchanged)
src/components/OperationTile/TokenTransferTile.test.tsx 11ms (unchanged)
src/components/OperationTile/TokenTransferTile.tsx 4ms (unchanged)
src/components/OperationTile/TransactionDirectionIcon.tsx 1ms (unchanged)
src/components/OperationTile/TransactionTile.test.tsx 7ms (unchanged)
src/components/OperationTile/TransactionTile.tsx 3ms (unchanged)
src/components/OperationTile/TzktLink.test.tsx 2ms (unchanged)
src/components/OperationTile/TzktLink.tsx 2ms (unchanged)
src/components/OperationTile/useIsOutgoingOperation.test.tsx 4ms (unchanged)
src/components/OperationTile/useIsOutgoingOperation.ts 2ms (unchanged)
src/components/OperationTile/useShowAddress.test.tsx 2ms (unchanged)
src/components/OperationTile/useShowAddress.tsx 1ms (unchanged)
src/components/PasswordInput.test.tsx 2ms (unchanged)
src/components/PasswordInput.tsx 3ms (unchanged)
src/components/PopoverMenu.tsx 1ms (unchanged)
src/components/PrettyNumber.test.tsx 1ms (unchanged)
src/components/PrettyNumber.tsx 1ms (unchanged)
src/components/ReceiveModal.tsx 2ms (unchanged)
src/components/RenameRemoveMenu.test.tsx 1ms (unchanged)
src/components/RenameRemoveMenu.tsx 1ms (unchanged)
src/components/Select.test.tsx 2ms (unchanged)
src/components/Select.tsx 2ms (unchanged)
src/components/SendButton.tsx 1ms (unchanged)
src/components/SendFlow/Batch/SignPage.test.tsx 2ms (unchanged)
src/components/SendFlow/Batch/SignPage.tsx 2ms (unchanged)
src/components/SendFlow/BatchModalBody.tsx 2ms (unchanged)
src/components/SendFlow/Beacon/BatchSignPage.tsx 2ms (unchanged)
src/components/SendFlow/Beacon/BeaconSignPage.tsx 1ms (unchanged)
src/components/SendFlow/Beacon/ContractCallSignPage.tsx 2ms (unchanged)
src/components/SendFlow/Beacon/DelegationSignPage.tsx 2ms (unchanged)
src/components/SendFlow/Beacon/Header.tsx 2ms (unchanged)
src/components/SendFlow/Beacon/TezSignPage.tsx 2ms (unchanged)
src/components/SendFlow/Beacon/UndelegationSignPage.tsx 2ms (unchanged)
src/components/SendFlow/Beacon/useSignWithBeacon.tsx 2ms (unchanged)
src/components/SendFlow/Delegation/FormPage.test.tsx 5ms (unchanged)
src/components/SendFlow/Delegation/FormPage.tsx 2ms (unchanged)
src/components/SendFlow/Delegation/SignPage.test.tsx 2ms (unchanged)
src/components/SendFlow/Delegation/SignPage.tsx 2ms (unchanged)
src/components/SendFlow/FormPageHeader.tsx 2ms (unchanged)
src/components/SendFlow/Multisig/SignPage.test.tsx 2ms (unchanged)
src/components/SendFlow/Multisig/SignPage.tsx 2ms (unchanged)
src/components/SendFlow/MultisigAccount/NameMultisigFormPage.test.tsx 6ms (unchanged)
src/components/SendFlow/MultisigAccount/NameMultisigFormPage.tsx 3ms (unchanged)
src/components/SendFlow/MultisigAccount/SelectApproversFormPage.test.tsx 12ms (unchanged)
src/components/SendFlow/MultisigAccount/SelectApproversFormPage.tsx 6ms (unchanged)
src/components/SendFlow/MultisigAccount/SignTransactionFormPage.test.tsx 2ms (unchanged)
src/components/SendFlow/MultisigAccount/SignTransactionFormPage.tsx 4ms (unchanged)
src/components/SendFlow/NFT/FormPage.test.tsx 5ms (unchanged)
src/components/SendFlow/NFT/FormPage.tsx 4ms (unchanged)
src/components/SendFlow/NFT/SignPage.test.tsx 3ms (unchanged)
src/components/SendFlow/NFT/SignPage.tsx 2ms (unchanged)
src/components/SendFlow/onSubmitFormActionHooks.tsx 3ms (unchanged)
src/components/SendFlow/OperationSignerSelector.test.tsx 2ms (unchanged)
src/components/SendFlow/OperationSignerSelector.tsx 1ms (unchanged)
src/components/SendFlow/SendNFTRecapTile.tsx 1ms (unchanged)
src/components/SendFlow/SignButton.tsx 4ms (unchanged)
src/components/SendFlow/SignPageFee.tsx 1ms (unchanged)
src/components/SendFlow/SignPageHeader.test.tsx 2ms (unchanged)
src/components/SendFlow/SignPageHeader.tsx 2ms (unchanged)
src/components/SendFlow/SuccessStep.tsx 1ms (unchanged)
src/components/SendFlow/Tez/FormPage.test.tsx 8ms (unchanged)
src/components/SendFlow/Tez/FormPage.tsx 3ms (unchanged)
src/components/SendFlow/Tez/SignPage.test.tsx 1ms (unchanged)
src/components/SendFlow/Tez/SignPage.tsx 2ms (unchanged)
src/components/SendFlow/Token/FormPage.test.tsx 6ms (unchanged)
src/components/SendFlow/Token/FormPage.tsx 4ms (unchanged)
src/components/SendFlow/Token/SignPage.test.tsx 3ms (unchanged)
src/components/SendFlow/Token/SignPage.tsx 2ms (unchanged)
src/components/SendFlow/Undelegation/FormPage.test.tsx 3ms (unchanged)
src/components/SendFlow/Undelegation/FormPage.tsx 2ms (unchanged)
src/components/SendFlow/Undelegation/SignPage.test.tsx 2ms (unchanged)
src/components/SendFlow/Undelegation/SignPage.tsx 2ms (unchanged)
src/components/SendFlow/utils.test.tsx 3ms (unchanged)
src/components/SendFlow/utils.tsx 7ms (unchanged)
src/components/SideNavbar.test.tsx 4ms (unchanged)
src/components/SideNavbar.tsx 4ms (unchanged)
src/components/Slider.tsx 1ms (unchanged)
src/components/SliderItem.tsx 1ms (unchanged)
src/components/SmallTab.tsx 1ms (unchanged)
src/components/TezRecapDisplay.tsx 1ms (unchanged)
src/components/TokenTile.test.tsx 2ms (unchanged)
src/components/TokenTile.tsx 1ms (unchanged)
src/components/TopBar.tsx 3ms (unchanged)
src/components/TruncatedTextWithTooltip.test.tsx 1ms (unchanged)
src/components/TruncatedTextWithTooltip.tsx 1ms (unchanged)
src/components/TzktLink.tsx 1ms (unchanged)
src/components/UpdateAppButton.tsx 1ms (unchanged)
src/components/useAccountsFilter.test.tsx 17ms (unchanged)
src/components/useAccountsFilter.tsx 3ms (unchanged)
src/components/useStepHistory.test.tsx 4ms (unchanged)
src/components/useStepHistory.tsx 1ms (unchanged)
src/e2e/features/multisig.feature 4ms (unchanged)
src/e2e/features/onboarding.feature 3ms (unchanged)
src/e2e/features/updates.feature 1ms (unchanged)
src/e2e/fixtures/backups/backedupAccountGroups.ts 2ms (unchanged)
src/e2e/fixtures/backups/V1Backup.json 1ms (unchanged)
src/e2e/fixtures/backups/V2Backup.json 295ms (unchanged)
src/e2e/helpers/AccountGroup.ts 7ms (unchanged)
src/e2e/pages/AccountsPage.ts 4ms (unchanged)
src/e2e/pages/CreateMultisigPage.ts 1ms (unchanged)
src/e2e/steps/hooks.ts 4ms (unchanged)
src/e2e/steps/multisig.ts 3ms (unchanged)
src/e2e/steps/onboarding.ts 7ms (unchanged)
src/e2e/steps/utils.ts 5ms (unchanged)
src/e2e/steps/world.ts 3ms (unchanged)
src/e2e/teardown.ts 0ms (unchanged)
src/e2e/utils.ts 3ms (unchanged)
src/GoogleAuth/getGoogleCredentials.ts 2ms (unchanged)
src/GoogleAuth/GoogleAuth.test.tsx 2ms (unchanged)
src/GoogleAuth/GoogleAuth.tsx 2ms (unchanged)
src/GoogleAuth/index.tsx 0ms (unchanged)
src/GoogleAuth/parseTorusRedirectParams.test.ts 2ms (unchanged)
src/GoogleAuth/parseTorusRedirectParams.ts 2ms (unchanged)
src/index.css 14ms (unchanged)
src/index.tsx 2ms (unchanged)
src/mocks/addressKind.ts 1ms (unchanged)
src/mocks/contacts.ts 1ms (unchanged)
src/mocks/dynamicModal.tsx 0ms (unchanged)
src/mocks/errorContext.ts 1ms (unchanged)
src/mocks/fa12Tokens.ts 2ms (unchanged)
src/mocks/fa2Tokens.ts 1ms (unchanged)
src/mocks/factories.ts 10ms (unchanged)
src/mocks/helpers.ts 2ms (unchanged)
src/mocks/mockMnemonic.ts 1ms (unchanged)
src/mocks/multisig.ts 2ms (unchanged)
src/mocks/nftTokens.ts 1ms (unchanged)
src/mocks/testUtils.tsx 3ms (unchanged)
src/mocks/toast.ts 1ms (unchanged)
src/mocks/tokens.ts 2ms (unchanged)
src/mocks/transfers.ts 1ms (unchanged)
src/mocks/tzktResponse.ts 4ms (unchanged)
src/mocks/umamiV1BackupFile.ts 1ms (unchanged)
src/multisig/contract.ts 14ms (unchanged)
src/multisig/decode/decodeLambda.test.ts 19ms (unchanged)
src/multisig/decode/decodeLambda.ts 6ms (unchanged)
src/multisig/decode/schemas.ts 5ms (unchanged)
src/multisig/decode/UnrecognizedMichelsonError.ts 0ms (unchanged)
src/providers/ReactQueryProvider.tsx 1ms (unchanged)
src/providers/ReduxStore.tsx 0ms (unchanged)
src/providers/UmamiTheme.tsx 0ms (unchanged)
src/react-app-env.d.ts 3ms (unchanged)
src/Router.tsx 3ms (unchanged)
src/setupTests.ts 3ms (unchanged)
src/style/colors.ts 1ms (unchanged)
src/style/theme.ts 2ms (unchanged)
src/style/theme/button.ts 4ms (unchanged)
src/style/theme/checkbox.ts 1ms (unchanged)
src/style/theme/drawer.ts 1ms (unchanged)
src/style/theme/list.ts 2ms (unchanged)
src/style/theme/modal.ts 2ms (unchanged)
src/style/theme/radio.ts 1ms (unchanged)
src/style/theme/tabs.ts 1ms (unchanged)
src/types/Account.ts 1ms (unchanged)
src/types/AccountOperations.ts 1ms (unchanged)
src/types/Address.ts 1ms (unchanged)
src/types/Contact.ts 0ms (unchanged)
src/types/Delegate.ts 0ms (unchanged)
src/types/Delegation.ts 1ms (unchanged)
src/types/Network.ts 1ms (unchanged)
src/types/Operation.test.ts 8ms (unchanged)
src/types/Operation.ts 6ms (unchanged)
src/types/SignerConfig.ts 1ms (unchanged)
src/types/Token.test.tsx 9ms (unchanged)
src/types/Token.ts 9ms (unchanged)
src/types/TokenBalance.test.ts 1ms (unchanged)
src/types/TokenBalance.ts 2ms (unchanged)
src/types/Transfer.ts 0ms (unchanged)
src/types/UmamiBackup.ts 0ms (unchanged)
src/utils/account/derivationPathUtils.test.tsx 1ms (unchanged)
src/utils/account/derivationPathUtils.ts 2ms (unchanged)
src/utils/account/makeMnemonicAccount.ts 1ms (unchanged)
src/utils/beacon/beacon.tsx 2ms (unchanged)
src/utils/beacon/BeaconPeers.test.tsx 7ms (unchanged)
src/utils/beacon/BeaconPeers.tsx 4ms (unchanged)
src/utils/beacon/BeaconProvider.tsx 1ms (unchanged)
src/utils/beacon/PermissionRequestModal.test.tsx 3ms (unchanged)
src/utils/beacon/PermissionRequestModal.tsx 3ms (unchanged)
src/utils/beacon/SignPayloadRequestModal.test.tsx 2ms (unchanged)
src/utils/beacon/SignPayloadRequestModal.tsx 2ms (unchanged)
src/utils/beacon/types.ts 0ms (unchanged)
src/utils/beacon/useHandleBeaconMessage.test.tsx 8ms (unchanged)
src/utils/beacon/useHandleBeaconMessage.tsx 3ms (unchanged)
src/utils/beacon/WalletClient.tsx 1ms (unchanged)
src/utils/crypto/AES_MODE.ts 0ms (unchanged)
src/utils/crypto/AES.test.ts 1ms (unchanged)
src/utils/crypto/AES.ts 2ms (unchanged)
src/utils/crypto/KDF.ts 2ms (unchanged)
src/utils/crypto/types.ts 1ms (unchanged)
src/utils/format.test.ts 1ms (unchanged)
src/utils/format.ts 1ms (unchanged)
src/utils/getErrorContext.test.ts 1ms (unchanged)
src/utils/getErrorContext.ts 1ms (unchanged)
src/utils/helpers.ts 0ms (unchanged)
src/utils/hooks/assetsHooks.test.ts 2ms (unchanged)
src/utils/hooks/assetsHooks.ts 5ms (unchanged)
src/utils/hooks/batchesHooks.ts 1ms (unchanged)
src/utils/hooks/beaconHooks.test.ts 3ms (unchanged)
src/utils/hooks/beaconHooks.ts 1ms (unchanged)
src/utils/hooks/contactsHooks.ts 1ms (unchanged)
src/utils/hooks/contactsUtils.test.ts 0ms (unchanged)
src/utils/hooks/contactsUtils.ts 1ms (unchanged)
src/utils/hooks/delegationHooks.ts 1ms (unchanged)
src/utils/hooks/getAccountDataHooks.test.ts 6ms (unchanged)
src/utils/hooks/getAccountDataHooks.ts 5ms (unchanged)
src/utils/hooks/multisigHooks.test.tsx 1ms (unchanged)
src/utils/hooks/multisigHooks.ts 1ms (unchanged)
src/utils/hooks/networkHooks.test.ts 2ms (unchanged)
src/utils/hooks/networkHooks.ts 1ms (unchanged)
src/utils/hooks/setAccountDataHooks.test.ts 8ms (unchanged)
src/utils/hooks/setAccountDataHooks.ts 4ms (unchanged)
src/utils/hooks/tokensHooks.test.ts 3ms (unchanged)
src/utils/hooks/tokensHooks.ts 1ms (unchanged)
src/utils/hooks/useAsyncActionHandler.test.ts 4ms (unchanged)
src/utils/hooks/useAsyncActionHandler.ts 2ms (unchanged)
src/utils/ledger/pk.ts 1ms (unchanged)
src/utils/mnemonic.test.ts 4ms (unchanged)
src/utils/mnemonic.ts 2ms (unchanged)
src/utils/multisig/fetch.test.ts 3ms (unchanged)
src/utils/multisig/fetch.ts 2ms (unchanged)
src/utils/multisig/helper.test.ts 3ms (unchanged)
src/utils/multisig/helpers.ts 2ms (unchanged)
src/utils/multisig/types.ts 1ms (unchanged)
src/utils/redux/hooks.ts 1ms (unchanged)
src/utils/redux/migrations.ts 2ms (unchanged)
src/utils/redux/persistor.ts 1ms (unchanged)
src/utils/redux/reducer.ts 1ms (unchanged)
src/utils/redux/slices/accountsSlice.test.ts 8ms (unchanged)
src/utils/redux/slices/accountsSlice.ts 4ms (unchanged)
src/utils/redux/slices/announcementSlice.test.ts 1ms (unchanged)
src/utils/redux/slices/announcementSlice.ts 1ms (unchanged)
src/utils/redux/slices/assetsSlice.test.ts 4ms (unchanged)
src/utils/redux/slices/assetsSlice.ts 4ms (unchanged)
src/utils/redux/slices/batches.test.ts 8ms (unchanged)
src/utils/redux/slices/batches.ts 3ms (unchanged)
src/utils/redux/slices/beaconSlice.test.ts 3ms (unchanged)
src/utils/redux/slices/beaconSlice.ts 2ms (unchanged)
src/utils/redux/slices/contactsSlice.test.ts 3ms (unchanged)
src/utils/redux/slices/contactsSlice.ts 1ms (unchanged)
src/utils/redux/slices/errorsSlice.test.ts 2ms (unchanged)
src/utils/redux/slices/errorsSlice.ts 1ms (unchanged)
src/utils/redux/slices/multisigsSlice.test.ts 3ms (unchanged)
src/utils/redux/slices/multisigsSlice.ts 2ms (unchanged)
src/utils/redux/slices/networks.test.ts 3ms (unchanged)
src/utils/redux/slices/networks.ts 2ms (unchanged)
src/utils/redux/slices/tokensSlice.ts 1ms (unchanged)
src/utils/redux/store.ts 0ms (unchanged)
src/utils/redux/thunks/changeMnemonicPassword.test.ts 3ms (unchanged)
src/utils/redux/thunks/changeMnemonicPassword.ts 2ms (unchanged)
src/utils/redux/thunks/checkAccountsAndUpsertContact.ts 1ms (unchanged)
src/utils/redux/thunks/estimateAndUpdateBatch.test.ts 2ms (unchanged)
src/utils/redux/thunks/estimateAndUpdateBatch.ts 1ms (unchanged)
src/utils/redux/thunks/renameAccount.test.ts 3ms (unchanged)
src/utils/redux/thunks/renameAccount.ts 1ms (unchanged)
src/utils/redux/thunks/secretKeyAccount.test.ts 3ms (unchanged)
src/utils/redux/thunks/secretKeyAccount.ts 2ms (unchanged)
src/utils/tezos/constants.ts 0ms (unchanged)
src/utils/tezos/estimate.test.ts 1ms (unchanged)
src/utils/tezos/estimate.ts 2ms (unchanged)
src/utils/tezos/execute.ts 1ms (unchanged)
src/utils/tezos/fakeSigner.test.ts 1ms (unchanged)
src/utils/tezos/fakeSigner.ts 1ms (unchanged)
src/utils/tezos/fetch.test.ts 18ms (unchanged)
src/utils/tezos/fetch.ts 7ms (unchanged)
src/utils/tezos/helpers.test.ts 6ms (unchanged)
src/utils/tezos/helpers.ts 5ms (unchanged)
src/utils/tezos/index.ts 0ms (unchanged)
src/utils/tezos/types.ts 1ms (unchanged)
src/utils/token/utils.test.ts 2ms (unchanged)
src/utils/token/utils.ts 1ms (unchanged)
src/utils/tzkt/helpers.test.ts 1ms (unchanged)
src/utils/tzkt/helpers.ts 1ms (unchanged)
src/utils/tzkt/types.ts 1ms (unchanged)
src/utils/useAssetsPolling.test.ts 2ms (unchanged)
src/utils/useAssetsPolling.ts 5ms (unchanged)
src/utils/useDeeplinkHandler.ts 2ms (unchanged)
src/utils/withTimeout.ts 1ms (unchanged)
src/views/addressBook/AddressBookView.test.tsx 2ms (unchanged)
src/views/addressBook/AddressBookView.tsx 1ms (unchanged)
src/views/addressBook/ContactTable.tsx 2ms (unchanged)
src/views/batch/AccountSmallTile.test.tsx 2ms (unchanged)
src/views/batch/AccountSmallTile.tsx 2ms (unchanged)
src/views/batch/BatchPage.test.tsx 4ms (unchanged)
src/views/batch/BatchPage.tsx 2ms (unchanged)
src/views/batch/BatchView.test.tsx 5ms (unchanged)
src/views/batch/BatchView.tsx 6ms (unchanged)
src/views/batch/OperationEstimationStatus.test.tsx 1ms (unchanged)
src/views/batch/OperationEstimationStatus.tsx 1ms (unchanged)
src/views/batch/OperationRecipient.test.tsx 1ms (unchanged)
src/views/batch/OperationRecipient.tsx 1ms (unchanged)
src/views/batch/OperationView.test.tsx 3ms (unchanged)
src/views/batch/OperationView.tsx 3ms (unchanged)
src/views/batch/tokenTitle.test.ts 1ms (unchanged)
src/views/batch/tokenTitle.ts 1ms (unchanged)
src/views/help/HelpView.test.tsx 1ms (unchanged)
src/views/help/HelpView.tsx 2ms (unchanged)
src/views/home/AccountGroupPopover.tsx 1ms (unchanged)
src/views/home/AccountListWithDrawer.tsx 3ms (unchanged)
src/views/home/AccountsList.test.tsx 8ms (unchanged)
src/views/home/AccountsList.tsx 6ms (unchanged)
src/views/home/DeriveAccountDisplay.tsx.tsx 2ms (unchanged)
src/views/home/DrawerTopButtons.tsx 1ms (unchanged)
src/views/home/HomeView.test.tsx 2ms (unchanged)
src/views/home/HomeView.tsx 1ms (unchanged)
src/views/home/SelectedAccountContext.ts 0ms (unchanged)
src/views/nfts/drawer/AttributesAccordionItem.test.tsx 2ms (unchanged)
src/views/nfts/drawer/AttributesAccordionItem.tsx 1ms (unchanged)
src/views/nfts/drawer/PropertiesAccordionItem.test.tsx 3ms (unchanged)
src/views/nfts/drawer/PropertiesAccordionItem.tsx 4ms (unchanged)
src/views/nfts/drawer/TagsSection.test.tsx 1ms (unchanged)
src/views/nfts/drawer/TagsSection.tsx 1ms (unchanged)
src/views/nfts/NFTCard.test.tsx 3ms (unchanged)
src/views/nfts/NFTCard.tsx 3ms (unchanged)
src/views/nfts/NFTDrawerBody.tsx 1ms (unchanged)
src/views/nfts/NFTDrawerCard.test.tsx 2ms (unchanged)
src/views/nfts/NFTDrawerCard.tsx 3ms (unchanged)
src/views/nfts/NFTGallery.test.tsx 2ms (unchanged)
src/views/nfts/NFTGallery.tsx 2ms (unchanged)
src/views/nfts/NFTsView.test.tsx 3ms (unchanged)
src/views/nfts/NftsView.tsx 2ms (unchanged)
src/views/nfts/SelectedNFTContext.ts 0ms (unchanged)
src/views/operations/OperationsView.test.tsx 3ms (unchanged)
src/views/operations/OperationsView.tsx 2ms (unchanged)
src/views/operations/useGetOperations.test.tsx 1ms (unchanged)
src/views/operations/useGetOperations.tsx 4ms (unchanged)
src/views/settings/DAppsDrawerCard.tsx 2ms (unchanged)
src/views/settings/ErrorLogsDrawerCard.tsx 3ms (unchanged)
src/views/settings/network/NetworkSettingsDrawerBody.test.tsx 1ms (unchanged)
src/views/settings/network/NetworkSettingsDrawerBody.tsx 2ms (unchanged)
src/views/settings/network/NetworkSettingsDrawerCard.tsx 1ms (unchanged)
src/views/settings/network/UpsertNetworkModal.test.tsx 4ms (unchanged)
src/views/settings/network/UpsertNetworkModal.tsx 3ms (unchanged)
src/views/settings/SettingsView.tsx 2ms (unchanged)
src/views/tokens/AccountTokens.test.tsx 2ms (unchanged)
src/views/tokens/AccountTokens.tsx 4ms (unchanged)
src/views/tokens/TokenNameWithIcon.test.tsx 1ms (unchanged)
src/views/tokens/TokenNameWithIcon.tsx 2ms (unchanged)
src/views/tokens/TokensPage.test.tsx 2ms (unchanged)
src/views/tokens/TokensPage.tsx 1ms (unchanged)
src/views/withSideMenu.tsx 0ms (unchanged)
src/WelcomeScreen.test.tsx 1ms (unchanged)
src/WelcomeScreen.tsx 2ms (unchanged)
tsconfig.e2e.json 1ms (unchanged)
tsconfig.json 0ms (unchanged)
➜  umami-v2 git:(fix-beacon-modal-opening) CSC_LINK=~/Documents/appleDeveloperExported.p12 CSC_KEY_PASSWORD=sergeythisisforyou1 APPLE_APP_SPECIFIC_PASSWORD='esvz-hvbw-elmo-aiyw' APPLE_ID=sergey.kintsel@trili.tech yarn electron:package:mac
  • electron-builder  version=24.13.3 os=23.2.0
  • loaded configuration  file=package.json ("build" field)
  • writing effective config  file=dist/builder-effective-config.yaml
  • packaging       platform=darwin arch=x64 electron=29.1.0 appOutDir=dist/mac-universal-x64-temp
  • downloading     url=https://github.com/electron/electron/releases/download/v29.1.0/electron-v29.1.0-darwin-x64.zip size=101 MB parts=8
  • downloaded      url=https://github.com/electron/electron/releases/download/v29.1.0/electron-v29.1.0-darwin-x64.zip duration=2.939s
  • packaging       platform=darwin arch=arm64 electron=29.1.0 appOutDir=dist/mac-universal-arm64-temp
  • downloading     url=https://github.com/electron/electron/releases/download/v29.1.0/electron-v29.1.0-darwin-arm64.zip size=95 MB parts=8
  • downloaded      url=https://github.com/electron/electron/releases/download/v29.1.0/electron-v29.1.0-darwin-arm64.zip duration=1.79s
  • packaging       platform=darwin arch=universal electron=29.1.0 appOutDir=dist/mac-universal
  • signing         file=dist/mac-universal/Umami.app platform=darwin type=distribution identity=D4C1D93EC0198A85168971FE720815BD0B731F07 provisioningProfile=none
  • notarization successful
  • building        target=DMG arch=universal file=dist/Umami-2.0.2-universal.dmg
  • building        target=macOS zip arch=universal file=dist/Umami-2.0.2-universal-mac.zip
  • Detected arm64 process, HFS+ is unavailable. Creating dmg with APFS - supports Mac OSX 10.12+
  • building block map  blockMapFile=dist/Umami-2.0.2-universal.dmg.blockmap
  • building block map  blockMapFile=dist/Umami-2.0.2-universal-mac.zip.blockmap
➜  umami-v2 git:(fix-beacon-modal-opening) open dist 
➜  umami-v2 git:(fix-beacon-modal-opening) glg
➜  umami-v2 git:(fix-beacon-modal-opening) ✗ gb
➜  umami-v2 git:(fix-beacon-modal-opening) ✗ gst 
On branch fix-beacon-modal-opening
Your branch is up to date with 'origin/fix-beacon-modal-opening'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   src/utils/beacon/BeaconPeers.tsx

no changes added to commit (use "git add" and/or "git commit -a")
➜  umami-v2 git:(fix-beacon-modal-opening) ✗ g
usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]
           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]
           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]
           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
           [--super-prefix=<path>] [--config-env=<name>=<envvar>]
           <command> [<args>]

These are common Git commands used in various situations:

start a working area (see also: git help tutorial)
   clone     Clone a repository into a new directory
   init      Create an empty Git repository or reinitialize an existing one

work on the current change (see also: git help everyday)
   add       Add file contents to the index
   mv        Move or rename a file, a directory, or a symlink
   restore   Restore working tree files
   rm        Remove files from the working tree and from the index

examine the history and state (see also: git help revisions)
   bisect    Use binary search to find the commit that introduced a bug
   diff      Show changes between commits, commit and working tree, etc
   grep      Print lines matching a pattern
   log       Show commit logs
   show      Show various types of objects
   status    Show the working tree status

grow, mark and tweak your common history
   branch    List, create, or delete branches
   commit    Record changes to the repository
   merge     Join two or more development histories together
   rebase    Reapply commits on top of another base tip
   reset     Reset current HEAD to the specified state
   switch    Switch branches
   tag       Create, list, delete or verify a tag object signed with GPG

collaborate (see also: git help workflows)
   fetch     Download objects and refs from another repository
   pull      Fetch from and integrate with another repository or a local branch
   push      Update remote refs along with associated objects

'git help -a' and 'git help -g' list available subcommands and some
concept guides. See 'git help <command>' or 'git help <concept>'
to read about a specific subcommand or concept.
See 'git help git' for an overview of the system.
➜  umami-v2 git:(fix-beacon-modal-opening) ✗ gco .
Updated 1 path from the index
➜  umami-v2 git:(fix-beacon-modal-opening) gco support-multipple-beacon-connections-to-the-same-app
Switched to branch 'support-multipple-beacon-connections-to-the-same-app'
Your branch is up to date with 'origin/support-multipple-beacon-connections-to-the-same-app'.
➜  umami-v2 git:(support-multipple-beacon-connections-to-the-same-app) glg
➜  umami-v2 git:(support-multipple-beacon-connections-to-the-same-app) git show

+      ).toEqual({
+        items: [
+          {
+            ...mockSecretKeyAccount(0),
+            curve: "ed25519",
+          },
+        ],
+      });
+    });
+  });
+});
diff --git a/src/utils/redux/migrations.ts b/src/utils/redux/migrations.ts
index 5b66e7f9..01da0807 100644
--- a/src/utils/redux/migrations.ts
+++ b/src/utils/redux/migrations.ts
@@ -16,6 +16,12 @@ export const mainStoreMigrations = {
       draft.announcement = announcementsInitialState;
     }),
   2: identity,
+  3: (state: any) =>
+    produce(state, (draft: any) => {
+      Object.entries(draft.beacon).forEach(([dAppId, connectionInfo]: [string, any]) => {
+        draft.beacon[dAppId] = { [connectionInfo.accountPkh]: connectionInfo.networkType };
+      });
+    }),
 } as any;

```

Reply:
nice.
```
+++ b/src/utils/redux/migrations.test.ts
@@ -0,0 +1,48 @@
+import { accountsMigrations, mainStoreMigrations } from "./migrations";
+import { mockImplicitAddress, mockSecretKeyAccount } from "../../mocks/factories";
+
+describe("migrations", () => {
+  describe("main", () => {
+    test("0", () => {
+      expect(mainStoreMigrations[0]({ multisigs: {} })).toEqual({ multisigs: { labelsMap: {} } });
+    });
+
+    test("1", () => {
+      expect(mainStoreMigrations[1]({})).toEqual({
+        announcement: {
+          html: "",
+          seen: true,
+        },
+      });
+    });
+
+    test("3", () => {
+      expect(
+        mainStoreMigrations[3]({
+          beacon: {
+            someDAppId: { accountPkh: mockImplicitAddress(0).pkh, networkType: "mainnet" },
+          },
+        })
+      ).toEqual({
+        beacon: {
+          someDAppId: { [mockImplicitAddress(0).pkh]: "mainnet" },
+        },
+      });
+    });
+  });
+
+  describe("accounts", () => {
+    test("2", () => {
+      expect(
+        accountsMigrations[2]({ items: [{ ...mockSecretKeyAccount(0), curve: undefined }] })
+      ).toEqual({
+        items: [
+          {
+            ...mockSecretKeyAccount(0),
+            curve: "ed25519",
+          },
+        ],
+      });
+    });
+  });
+});
diff --git a/src/utils/redux/migrations.ts b/src/utils/redux/migrations.ts
index 5b66e7f9..01da0807 100644
--- a/src/utils/redux/migrations.ts
+++ b/src/utils/redux/migrations.ts
@@ -16,6 +16,12 @@ export const mainStoreMigrations = {
       draft.announcement = announcementsInitialState;
     }),
   2: identity,
+  3: (state: any) =>
+    produce(state, (draft: any) => {
+      Object.entries(draft.beacon).forEach(([dAppId, connectionInfo]: [string, any]) => {
+        draft.beacon[dAppId] = { [connectionInfo.accountPkh]: connectionInfo.networkType };
+      });
+    }),
 } as any;

```

Reply:
also, next time you could use not `git push --force` but `git push --force-with-lease` to prevent it. [here's an explanation](https://stackoverflow.com/a/52823955) what's the difference between the two

Reply:
Thank you for the code - added.

Will use ```--force-with-lease```in the future